soql-stdlib/src/main/scala/com/socrata/soql/functions/SoQLFunctions.scala

```diff
@@ -453,6 +453,9 @@ object SoQLFunctions {
   val FloatingTimeStampExtractWoy = mf("floating timestamp extract week of year", FunctionName("date_extract_woy"), Seq(SoQLFloatingTimestamp), Seq.empty, SoQLNumber)(
```

Comment:
I might change the string to be something like `floating timestamp extract ISO week of year` and/or add a comment that `woy` is ISO week just to leave a few crumbs to make that more obvious to folks in the future despite the function not being named `iso`.